code/__HELPERS/unsorted.dm

```diff
@@ -1448,3 +1448,40 @@ B --><-- A
 	if(orbiting)
 		loc = get_turf(orbiting)
 		orbiting = null
+
+
+//Center's an image.

```

Comment:
Criticise not my english, I'm lazy, and this like this are ingrained errors from long ago.
I sometimes randomly capitalise words too, that's another one.

tl;dr shush :S